### PR TITLE
Fix submission workflow, Add edit prev submission capability

### PR DESCRIPTION
**Pull request by birdbrained**

### CODE REVIEW

File: app/components/item-detail-page/component.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export default Ember.Component.extend({
     constructedItem: null,
     didReceiveAttrs () {
         const type = this.get('item.type');
+        console.log(type)
```
Comment (cameronblandford) [inline app/components/item-detail-page/component.js]: did you mean to commit this?

File: app/components/item-detail-page/template.hbs [hunk 2]
```diff
@@ -32,21 +32,16 @@
 <div class="container">
     <div class="row p-t-lg">
             <div class="col-sm-8">
-                {{#if constructedItem.viewContent.file.visible}}
-                    {{#if constructedItem.viewContent.file.loaded}}
                         <div>
-                            {{file-renderer download=constructedItem.viewContent.file.value width="99%" height="1000" allowfullscreen=true}}
+                            {{file-renderer download=item.fileLink width="99%" height="1000" allowfullscreen=true}}
```
Comment (cameronblandford) [inline app/components/item-detail-page/template.hbs]: :+1:

File: app/components/meeting-submit/component.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ export default Ember.Component.extend({
     description: 'Submit',
 
     parameters: {},
-
+/*
```
Comment (cameronblandford) [inline app/components/meeting-submit/component.js]: were you able to get rid of the observers?

File: app/components/meeting-submit/component.js [hunk 4]
```diff
@@ -97,16 +101,14 @@ export default Ember.Component.extend({
 
             // TODO: REPLACE THESE WITH REAL WIDGETS
             item.set('metadata', '{}');
-            item.set('source_id', 'xrfye');
```
Comment (cameronblandford) [inline app/components/meeting-submit/component.js]: :+1:

File: parameter-alias.js [hunk 5]
```diff
@@ -1,21 +0,0 @@
-import DS from 'ember-data';
```
Comment (cameronblandford) [inline parameter-alias.js]: glad to see this go tbh

**Comment by cameronblandford** [comment]
What else needs to be changed before this is good to go, if anything?

File: app/models/widget.js [hunk 6]
```diff
@@ -33,28 +36,19 @@ export default Model.extend({
...
-                    this.set('parameters', resolved);
-                });
-            });
-            return this.get('_parameters');
-        },
-        set(key, value) {
-            this.set('_parameters', value);
-            return value;
+    caseParameters: Ember.computed('caxe.activeCase.parameters.@each', function() {
+        console.log('New Parameter!')
```
Comment (cameronblandford) [inline app/models/widget.js]: should this be kept?

**Comment by cameronblandford** [comment]
It might be worth taking some time to remove a lot of the commented out code. it's saved in the git history 👍

File: app/components/item-detail-page/component.js (hunk 1 above)
Comment (birdbrained) [reply app/components/item-detail-page/component.js]: nope

File: app/models/widget.js (hunk 6 above)
Comment (birdbrained) [reply app/models/widget.js]: yeah we need this

Comment (birdbrained) [reply app/models/widget.js]: oh wait were talking about the log

File: parameter-alias.js (hunk 5 above)
Comment (birdbrained) [reply parameter-alias.js]: this was never doing anything lol its in the root dir

Comment (birdbrained) [reply parameter-alias.js]: alias still needs to exist - it's in `app/models`

File: app/components/item-detail-page/component.js (hunk 1 above)
Comment (birdbrained) [reply app/components/item-detail-page/component.js]: nope...

File: parameter-alias.js (hunk 5 above)
Comment (cameronblandford) [reply parameter-alias.js]: thaaaaat makes more sense

File: app/models/widget.js (hunk 6 above)
Comment (cameronblandford) [reply app/models/widget.js]: yep, log!

File: app/components/file-uploader/component.js [hunk 7]
```diff
@@ -7,33 +7,20 @@ export default Ember.Component.extend({
 
     parameters: {},
 
-    fileNameObserver: Ember.observer('widget.parameters.fileName', function() {
```
Comment (cameronblandford) [inline app/components/file-uploader/component.js]: nice!

File: app/components/meeting-submit/component.js [hunk 8]
```diff
@@ -24,69 +24,19 @@ export default Ember.Component.extend({
 
     parameters: {},
 
-    typeObserver: Ember.observer('widget.parameters', 'widget.parameters.type.value', function() {
```
Comment (cameronblandford) [inline app/components/meeting-submit/component.js]: thank goodness :')

File: app/models/widget.js (hunk 6 above)
Comment (cameronblandford) [reply app/models/widget.js]: still need to remove this

File: app/routes/collection/editItem.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+        const collectionType = this.modelFor('collection').get('collectionType');
+        let submissionFormId = {
+            'meeting': 1,
+            'appendix': 2,
+            'dataset': 14
+        }[collectionType];
+        //const caxe = this.store.createRecord('case');
+        return this.store.findRecord("case", '464').then(caxe => {
+            this.set('caxe.activeCase', caxe);
+            //return this.store.findRecord('workflow', submissionFormId)
```
Comment (cameronblandford) [inline app/routes/collection/editItem.js]: remove?

**Comment by cameronblandford** [comment]
this looks real good!

**Pull request merged**